nodejs12.x/cookiecutter-typescript-app-template/{{cookiecutter.project_name}}/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "web_template",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "fast-compile": "tsc",
+    "compile": "eslint . --ext .ts && tsc",
+    "lint": "eslint . --ext .ts",
+    "test": "eslint . --ext .ts && tsc && mocha",
+    "fast-test": "mocha"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
What should I do with the `author` and `license` fields?

Reply:
Looking at the current templates it is always `SAM CLI` and `MIT`, I will try to find one example from the community and update here.